### ISSUE TITLE
type mismatch: got <Packet, string>

### ISSUE THREAD

**Issue by TensorTom** [status-im/nim-json-rpc#74]
Trying to use rpcclient. Got:

```
/nim-json-rpc/json_rpc/clients/websocketclient.nim(35, 48) template/generic instantiation of `async` from here
/nim-json-rpc/json_rpc/clients/websocketclient.nim(40, 16) Error: type mismatch: got <Packet, string>
but expected one of: 
proc `==`[T: tuple |
    object](x, y: T): bool
  first type mismatch at position: 2
  required type for y: T: tuple or object
  but expression '""' is of type: string
47 other mismatching symbols have been suppressed; compile with --showAllMismatches:on to see them

expression: value == ""
```

Nim Compiler Version 1.0.6 [Linux: amd64]

**Comment by jangko** [comment]
this is because nim-json-rpc is using [Tormund/news](https://github.com/Tormund/news) v0.2 while your installed news is v0.3

**Comment by jangko** [comment]
fixed by #103, closing.
